modules/apps/object/object-api/src/main/java/com/liferay/object/entry/util/ObjectEntryThreadLocalUtil.java

```diff
@@ -12,14 +12,14 @@
  * details.
  */
 
-package com.liferay.object.internal.entry.util;
+package com.liferay.object.entry.util;
 
 import com.liferay.petra.lang.CentralizedThreadLocal;
 
 /**
  * @author Marcela Cunha
  */
-public class ObjectEntryThreadLocal {
+public class ObjectEntryThreadLocalUtil {
```

Comment:
I approved because this comment is only related to the code already reviewed..

Reply:
Yes, i don't know too. But, i think that is because the class was move to other package. What do you think rename to `ObjectEntryThreadLocal` only @carolmariaabb ?

Reply:
Perfect! I will change the class name. Thank you @carolmariaabb